cylc/flow/option_parsers.py

```diff
@@ -440,6 +445,13 @@ def add_std_options(self):
                 ),
                 action="store", default=None, dest="templatevars_file")
 
+        if self.revalidate:
+            self.add_std_option(
+                '--revalidate',
+                help="Get template variables from prevous workflow run.",
+                action='store_true', default=False
+            )
```

Comment:
Were we going to infer this option if the provided workflow is a run directory?

Reply:
Yes